docs/web-apps/automated-testing/electron.md

```diff
@@ -0,0 +1,76 @@
+---
+id: electron
+title: Electron Support on Sauce Labs [Beta]
+sidebar_label: Electron
```

Comment:
Just for consistency with the other frameworks:
```suggestion
sidebar_label: Using Electron
```

Else you'll end up with a redundant menu that looks like this
```
Electron
  - Electron
```